CONTRIBUTIONS.md

```diff
@@ -0,0 +1,21 @@
+# Contributions
+
+## If you are interested in contributing, here are some ground rules:
+* Ideally, [talk to us](https://forum.unity.com/forums/input-system.103/) before doing the work -- we appreciate contributions, but we might already be working on the same thing, or we might have different opinions on how it should be implemented.
+
+## All contributions are subject to the [Unity Contribution Agreement(UCA)](https://unity3d.com/legal/licenses/Unity_Contribution_Agreement)

```

Comment:
Does this come from Legal? Its worth getting them to check and make sure the wording is correct.

Reply:
This one is copied from the various other CONTRIBUTIONS.md we have. Probably a bit cheap on my end but with some of that stuff already on trunk and with the doc itself up on unity3d.com/legal, I just told myself "this is already properly vetted"  :)